sirepo/raydata/scan_monitor.py

```diff
@@ -171,6 +162,38 @@ def statuses_for_scans(cls, catalog_name, rduids):
             .all()
         )
 
+    @classmethod
+    def _db_upgrade(cls):
+        def _fixup_running_statuses():
+            for x in cls.session.query(cls).filter(
+                cls.status.in_(_AnalysisStatus.NON_STOPPED)
+            ):
+
+                cls.set_scan_status(
+                    PKDict(rduid=x.rduid, catalog_name=x.catalog_name),
+                    _AnalysisStatus.ERROR,
+                )
+
+        def _rduid_in_schema():
+            global engine
```

Comment:
Doesn't need to be global. You only need to use global if you are assigning to it.

Reply:
897e04f358c990b4061a172752be22c4121e4e69